legend-engine-xt-serviceStore-executionPlan/src/test/java/org/finos/legend/engine/plan/execution/stores/service/securitySchemes/TestApiKeyAuthenticationWithApiKeySecurityScheme.java

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.service.securitySchemes;
+
+import org.finos.legend.engine.plan.execution.stores.service.utils.ServiceStoreTestSuite;
+import org.finos.legend.engine.plan.execution.stores.service.utils.ServiceStoreTestUtils;
+import org.finos.legend.engine.protocol.pure.v1.model.executionPlan.SingleExecutionPlan;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.finos.legend.engine.plan.execution.stores.service.utils.ServiceStoreTestUtils.buildPlanForQuery;
+import static org.finos.legend.engine.plan.execution.stores.service.utils.ServiceStoreTestUtils.executePlan;
+
+public class TestApiKeyAuthenticationWithApiKeySecurityScheme extends ServiceStoreTestSuite
+{
+    private static String pureGrammar;
+
+    @BeforeClass
+    public static void setup()
+    {
+        setupServer("securitySchemes");
```

Comment:
Lets please follow the conventions of other tests, everything other than connection should be in resource file.
Also instead of string replace you can use varaible port directly, right?

Reply:
service store moved to resource file.
I am keeping the variable port because thats how we are doing it in other tests also.